packages/components/src/mobile/gradient/index.native.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { View, Platform } from 'react-native';
+import RNLinearGradient from 'react-native-linear-gradient';
```

Comment:
Were you able to test using `react-native-svg`'s Linear Gradient? If it works can you please open an issue to remove `react-native-linear-gradient` in favor of the other one? Having one less library to maintain would be awesome :D

Reply:
Thanks for reminding me about that! I would like to open a separate issue to track and check if we can get rid of `rn-linear-gradient` in favour or `LinearGradient` from `rn-svg`. Maybe I will even check that in the second PR related to gradients 🤔

Reply:
https://github.com/wordpress-mobile/gutenberg-mobile/issues/2308